client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -171,6 +197,19 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
                     <ButtonGroup align="inline">
                         <FilterSubnavDropdown viewSize={this.state.viewSize} />
                     </ButtonGroup>
+                    {this.props.activefilter === 'EVENTS' ? ' ' : (
+                        <div>
+                          All items and Coverages assigned to :
```

Comment:
`gettext` is missing.
This label is also very long, and would not work on smaller screens.